src/client/types.ts

```diff
@@ -1,4 +1,6 @@
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
+import { Calculation } from "pluralistic";
```

Comment:
A less generic name here would help readers unfamiliar with the code to understand what's going on.